src/icon-packs.ts

```diff
@@ -79,6 +79,14 @@ const iconPacks = {
     downloadLink:
       'https://github.com/FlorianWoelki/obsidian-iconize/raw/main/iconPacks/rpg-awesome.zip',
   },
+  /** @source https://coolicons.cool/ */
+  coolicons: {
+    name: 'coolicons',
+    displayName: 'Coolicons',
+    path: '',
+    downloadLink:
+      'https://github.com/FlorianWoelki/obsidian-iconize/raw/main/iconPacks/coolicons.zip',
```

Comment:
You can probably get this directly from the official github release right?
ref: https://github.com/krystonschwarze/coolicons/releases/tag/v4.1

Reply:
Absolutely ! I thought it wouldn't work due to a structure with subfolders but it does :+1: